Libraries/Components/WebView/WebView.ios.js

```diff
@@ -311,6 +311,16 @@ class WebView extends React.Component {
     scalesPageToFit: PropTypes.bool,
 
     /**
+     * Function that is invoked when a defined URL-scheme has been rejected.
```

Comment:
This comment doesn't add any info to the name `onUrlSchemeRejected` below.

What does "rejected" mean?